observer/lib/telemetry.js

```diff
@@ -6,7 +6,7 @@ const debug = createDebug('spark:observer:telemetry')
 export const createInflux = token => {
   const influx = new InfluxDB({
     url: 'https://eu-central-1-1.aws.cloud2.influxdata.com',
-    // bucket permissions: spark-evaluate:read spark-observer:write station-machines:read
+    // bucket permissions: spark-evaluate:read spark-observer:write station:read
```

Comment:
Does this mirror the permissions the actual token has?

Reply:
Yes, are currently using token with those permissions. 

In the #311 we've introduced query that queries `station-machines` bucket. In the follow-up PR #328 we've changed the bucket we query but this comment was not updated.